domain/utxoindex/utxoindex.go

```diff
@@ -56,6 +62,11 @@ func (ui *UTXOIndex) Reset() error {
 		return err
 	}
 
+	err = ui.store.intalizeCirculatingSompiSupply() //calculate sompi supply from database before adding virtuals
```

Comment:
The DB here is always empty. You can change the comment to "calling intalizeCirculatingSompiSupply so the `circulatingSupplyKey` key will be set"

Reply:
intalizeCirculatingSompiSupply actually goes through the database and adds together all amounts found in it (also adds the key). does this even have any use? i see we delete the whole db a few lines up and fill it with virtuals afterwards. so wondering why i did this.

that being said if looping through an empty database is no overhead i can just leave it I suppose.

will change the comment